styx-common/src/test/java/com/spotify/styx/util/TimeUtilTest.java

```diff
@@ -21,12 +21,16 @@
 package com.spotify.styx.util;
 
 import static com.spotify.styx.util.TimeUtil.addOffset;
-import static com.spotify.styx.util.TimeUtil.instancesInRange;
+import static com.spotify.styx.util.TimeUtil.instantsInRange;
+import static com.spotify.styx.util.TimeUtil.instantsInReversedRange;
 import static com.spotify.styx.util.TimeUtil.isAligned;
 import static com.spotify.styx.util.TimeUtil.lastInstant;
 import static com.spotify.styx.util.TimeUtil.nextInstant;
+import static java.time.Instant.parse;
```

Comment:
This static import produces a lot of noise in this file, makes it harder to check what actually changed.

Reply:
Although the github diff highlights the actual changes, so no big deal